### PR TITLE
Add migration for R 4.1

### PR DESCRIPTION
**Pull request by xhochy**
FYI @conda-forge/r @conda-forge/r-base @conda-forge/core 

I would start this tomorrow morning CEST and would babysit it during the day to see if there are any blockers and likely increase the PR rate during the day.

Note that https://github.com/conda-forge/r-base-feedstock/pull/173 is still open but expected to be merged soon.

### CODE REVIEW

File: recipe/migrations/r410.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+migrator_ts: 1621536224
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  automerge: true
+
```
Comment (beckermr) [inline recipe/migrations/r410.yaml:10]: Add the long-term key for this one. See the other long running migrators like aarch

**Review by beckermr — CHANGES_REQUESTED** [review]

File: recipe/migrations/r410.yaml (hunk 1 above)
Comment (beckermr) [reply recipe/migrations/r410.yaml:10]: `longterm: True`

File: recipe/migrations/r410.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  # slow start, increase to 20/50 later
+  pr_limit: 2
+
```
Comment (beckermr) [inline recipe/migrations/r410.yaml:13]: @isuruf should we use a key add migrator here and keep the bump number to zero?

**Review by beckermr — CHANGES_REQUESTED** [review]
I have one more question.

File: recipe/migrations/r410.yaml (hunk 2 above)
Comment (isuruf) [reply recipe/migrations/r410.yaml:13]: Key add migrator is more expensive because we'll be keeping 3.6.
Keeping bump number to zero is good

Comment (beckermr) [reply recipe/migrations/r410.yaml:13]: Right. We may want to keep 3.6 for a while when this is running and then move it all at once.

Comment (xhochy) [reply recipe/migrations/r410.yaml:14]: This should be a fairly quick migration (in contrast to a new Python version) keeping 3.6 shouldn't be necessary. We didn't do it the last time and I don't think we ran into any issues.

Comment (beckermr) [reply recipe/migrations/r410.yaml:14]: OK. I will resolve this comment then.

**Comment by xhochy** [comment]
@beckermr Good to go now for tomorrow / later?

**Review by beckermr — APPROVED** [review]

**Pull request merged**